docs/index.md

```diff
@@ -1,19 +1,34 @@
 ---
 layout: page
-title: ClinicConnectSystem
+title: ClinicConnect
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
 ![Ui](images/Ui.png)
 
-**ClinicConnectSystem is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+# ***ClinicConnect* - The *ideal* patient management and appointment tracker system for health-screening clinics.**
+
+> *Tired* of sorting through dozens of files just to find the details of a patient coming for a health screening later?  
+>  
+> *Tired* of how relevant patient details such as allergies, blood types, health conditions are scattered all over different systems?  
+>  
+> *Tired* of manually scanning each individual appointment for the next week to ensure sufficient health screening materials for those health screenings?  
+
+Try out **ClinicConnect**, our revolutionary system that integrates **patient management** and **appointment tracking** into simple, cohesive package.
```

Comment:
Perhaps it could be:
"and appointment tracking into one simple, cohesive package."

Reply:
Nice catch, thanks!